### PR TITLE
add-fonts-size-and-line-styles

### PR DESCRIPTION
**Pull request by thanhnguyennguyen**
Hi @winminsoe ,
Could you please arrange your time help me review this pull request
Thanks
---------------
add line styles and font styles for desktop
usage example of line style:

>   border-top: $lineThin $lineSolid $lineDark;

### CODE REVIEW

File: _variables.scss [hunk 1]
```diff
@@ -34,3 +35,20 @@ $fontsizeL:      16px;
 $fontsizeXL:     18px;
 $fontsizeXXL:    22px;
 $fontsizeXXXL:   24px;
+
+
+// Various lines
+// can be used as borders, underlines, separators
+// https://projects.invisionapp.com/share/8CAS4KBDB#/screens/82068093_StyleguideM1-VisualElements-Lines
+
+// line weigh
```
Comment (winminsoe) [inline _variables.scss]: typo

File: _variables.scss [hunk 2]
```diff
@@ -34,3 +35,20 @@ $fontsizeL:      16px;
...
+$lineThin:        1px;
+$lineThick:       2px;
+
+// line style
+$lineDotted:      dotted;
```
Comment (winminsoe) [inline _variables.scss]: do we really need it as variable? as it's already in css variable.

Comment (thanhnguyennguyen) [reply _variables.scss]: we want to centralize them so that we easily update in the future
cc: @ameenakbar90210

Comment (winminsoe) [reply _variables.scss]: we should not do it. otherwise, we are adding all the css variables in here.

Comment (thanhnguyennguyen) [reply _variables.scss]: can you advise me a better solution to centralize these line styles?
https://projects.invisionapp.com/share/8CAS4KBDB#/screens/82068093_StyleguideM1-VisualElements-Lines

Comment (winminsoe) [reply _variables.scss]:
```
$primaryLine:      dotted;
```
is more meaningful if we have to do.

File: _variables.scss [hunk 3]
```diff
@@ -34,3 +47,16 @@ $fontsizeL:      16px;
...
+// can be used as borders, underlines, separators
+// https://projects.invisionapp.com/share/8CAS4KBDB#/screens/82068093_StyleguideM1-VisualElements-Lines
+
+// line weight
+$lineThin:        1px;
+$lineThick:       2px;
+
+// line color
+$lineLight:       #F4F4F4;
+$lineDark:        #DFDFDF;
```
Comment (winminsoe) [inline _variables.scss]: any reason why these two variable has duplicated values? https://github.com/thanhnguyennguyen/global-scss/blob/6284fb469bce2cca8bcdda8a308bab891e6a9eba/_variables.scss#L7-L8

better with 
```
$lineColorLight: $colorLightSecondary
```

**Pull request merged**